hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -270,6 +270,7 @@ private OzoneConsts() {
   public static final String SRC_KEY = "srcKey";
   public static final String DST_KEY = "dstKey";
   public static final String USED_BYTES = "usedBytes";
+  public static final String USED_NAMESPACE = "usedNamespace";
```

Comment:
Seems we don't use this variable, 
Could you be so kind as to let me know its usage if I miss something.

Reply:
Indeed it is not used. Removed this constant and we can add it in the future when there is a need.